OSPSuite.Presentation.Importer/Core/DataFormat/DataFormat_Nonmem.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OSPSuite.Presentation.Importer.Core.DataFormat
+{
+   public class DataFormat_Nonmem : AbstractColumnsDataFormat
+   {
+      public override string Name => "Nonmem Oriented";
+
+      public override string Description => "https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/797";
+
+      private int _lloq_index = -1;
+
+      protected override void extractGeneralParameters(List<string> keys, IUnformattedData data)
+      {
+         base.extractGeneralParameters(keys, data);
+         var lloq = data.GetHeaders().FirstOrDefault(h => h.ToUpper() == "LLOQ");
```

Comment:
We are doing this type of stuff so many times
data.GetHeaders... find t o upper
I think the UnformattedData could have some helper function in it to simplify some of the code.
Also why do we have an interface for IUnformattedData?

Reply:
Done